modules/admin_manual/pages/installation/quick_guides/ubuntu_20_04.adoc

```diff
@@ -60,12 +60,13 @@ apt install -y \
   wget
 ----
 
-Note : php 7.4 is default version installable with Ubuntu 20.04
+Note : php 7.4 is the default version installed with Ubuntu 20.04
```

Comment:
my only objection is that "installed" suggests that php 7.4 comes with Ubuntu 20.04 preinstalled.

Reply:
sorry, you are correct. `php` is not pre-installed:
```
$ php
Command 'ph' not found, but can be installed with:

sudo apt install php7.4-cli
```

I will revert back to "installable"